plsync/planetlab/model.py

```diff
@@ -577,7 +596,7 @@ def __init__(self, **kwargs):
             if type(kwargs['ipv6']) == str:
                 kwargs['ipv6'] = "all"
             elif type(kwargs['ipv6']) == type([]):
-                domain = '.measurement-lab.org'
+                domain = '.' + MLAB_ORG_DOMAIN
```

Comment:
Since this assignment only serves to add a dot, would it be more concise to simply remove ir and change the following line to something like the following?: [ h+'.'+MLAB_ORG_DOMAIN for h ...]